lua/lvim/config/settings.lua

```diff
@@ -51,6 +51,23 @@ M.load_options = function()
 
   vim.opt.shortmess:append "c"
 
+  -- Defaults:
+  --   Neovim: !,'100,<50,s10,h
+  --
+  -- - ! save/restore global variables (only all-uppercase variables)
+  -- - '100 save/restore marks from last 100 files
+  -- - <50 save/restore 50 lines from each register
+  -- - s10 max item size 10KB
+  -- - h do not save/restore 'hlsearch' setting
+  --
+  -- Our overrides (credit wincent):
+  -- - '50 save/restore marks from last 100 files
+  -- - <0 don't save registers
+  -- - f0 don't store file marks
+  -- - n: store in $LUNARVIM_CACHE_DIR/lvim.shada
+  --
+  vim.opt.shada = "'50,<0,f0,n" .. utils.join_paths(get_cache_dir(), "lvim.shada")
```

Comment:
Instead of using the `n` option, I would recommend setting `shadafile` instead since it overrides the `n` option
`:help sdf`

Reply:
I decided to use this option and leave the rest as default, since they have different values on Windows and I don't feel like digging through that.